code-execution/src/main/java/com/dojo/codeexecution/service/GitManager.java

```diff
@@ -81,4 +86,27 @@ private String getRepositoryNameByOwner(String username, String game) throws IOE
     private GHUser getGitHubUser(String username) throws IOException {
         return gitHub.getUser(username);
     }
+
+    public String deleteReposForParticularGame(String game) throws IOException {
+        GHMyself account = gitHub.getMyself();
+        Collection<GHRepository> listOfAllRepos = account.getAllRepositories().values();
+        List<String> nameOfDeletedRepos = new CopyOnWriteArrayList<>();
```

Comment:
deletedRepos would be enough for naming the list or deletedReposNames is also acceptable